### DISCUSSION TITLE
Add Option to Left/Right/Center Align Metrics Panel Value

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#16540] [Feature Requests]
This value should be able to be left aligned

![CleanShot 2022-11-20 at 16 03 49](https://user-images.githubusercontent.com/522079/202926018-70c3886a-d4e4-480c-a0dc-264b9f1bd350.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
